tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -57,7 +57,7 @@ describe('Acceptance: brocfile-smoke-test', function() {
       if (isExperimentEnabled('MODULE_UNIFICATION')) {
         expected = 'window.EmberENV = {"asdflkmawejf":";jlnu3yr23"};';
       } else {
-        expected = 'window.EmberENV = {"asdflkmawejf":";jlnu3yr23","_JQUERY_INTEGRATION":true};';
+        expected = 'window.EmberENV = {"asdflkmawejf":";jlnu3yr23","_JQUERY_INTEGRATION":false};';
```

Comment:
Not sure why there were separate assertions for MU, but I think this should be a single assertion now, i.e. the `if` can be removed. At least the MU-enabled test scenarios seem to fail because of this now...

Reply:
It looks like the CI is going to pass 😅

Reply:
I've done what you recommended 👍
Thank you

Reply:
whoo it's green
sorry for the late reply